scripts/qa_crowbarsetup.sh

```diff
@@ -7,11 +7,14 @@ shopt -s extglob
 
 : ${SCRIPTS_DIR:=$(dirname $(readlink -e $BASH_SOURCE))}
 scripts_lib_dir=${SCRIPTS_DIR}/lib
-common_scripts="mkcloud-common.sh"
+common_scripts="mkcloud-common.sh qa_crowbarsetup-help.sh"
 for script in $common_scripts; do
     source ${scripts_lib_dir}/$script
 done
 
+# not being sourced from mkcloud is a feature
+is_onhost && complain 9 "qa_crowbarsetup.sh should not be sourced within mkcloud. Use the common files for shared functions."
```

Comment:
This is still cryptic for someone who doesn't understand the issue.  Put yourself in their shoes :)  They will ask "Which common files?  Which shared functions?  What are you talking about?  How the hell do I fix this?"

IIUC, this can only happen if an old mkcloud accidentally uses a newer `qa_crowbarsetup.sh`, right?  So a much more helpful message would be to simply explain that.  Then they can easily understand that they need to update their `mkcloud`.